src/__test__/Calculator.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+describe('Calculator component test', () => {
```

Comment:
- [ ] `required`
- [ ] Good job so far!! 🥂 
- [ ] However, I'm failing to run the test or start the project
- [ ] see the screenshot below and the highlighted error

`error screenshot`
![image](https://user-images.githubusercontent.com/66230306/207984415-9a02de4e-13fd-4dd9-9bf5-186b22325a59.png)